js/ui/range_slider.js

```diff
@@ -297,6 +307,20 @@ const RangeSlider = Slider.inherit({
         this.option('value', [start, end]);
     },
 
+    _getActualStartValue: function() {
+        if(!this._actualValue) {
+            return undefined;
```

Comment:
let's return `this.option('start')` here to reduce copypaste in all such places.